packages/itwin/breakdown-trees/src/Views/SpatialContainmentTree.tsx

```diff
@@ -101,11 +101,11 @@ export const SpatialContainmentTree: React.FC<SpatialContainmentTreeProps> = (pr
 
     populateContextAndOptionMenuItems(treeName, functionIconMapperInner, optionItemsInner, dataProvider, spatialRules, props.eventHandlers, props.clipHeight, props.clipAtSpaces);
 
-    const displayGuidHandlerInner = new GenericOptionItemHandler("Show Guids", BreakdownTrees.translate("contextMenu.showGuids"), "icon-label", () => props.displayGuids, props.setIsDisplayGuids);
+    const displayGuidHandlerInner = new GenericOptionItemHandler("Show Guids", BreakdownTrees.translate("contextMenu.showGuids"), "icon - label", () => props.displayGuids, props.setIsDisplayGuids);
```

Comment:
why are you adding spaces to every icon string?

Reply:
i dont know why that happened, it maybe was a linter thing? i didnt do that myself

Reply:
changed